server/game/cards/01_SOR/units/BodhiRook.ts

```diff
@@ -0,0 +1,33 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class BodhiRook extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '7257556541',
+            internalName: 'bodhi-rook#imperial-defector'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'Look at an opponent\'s hand and discard a non-unit card from it.',
+            immediateEffect: AbilityHelper.immediateEffects.sequential([
+                AbilityHelper.immediateEffects.lookAt((context) => ({
+                    target: context.player.opponent.hand.sort((a, b) => a.title.localeCompare(b.title)),
+                    sendChatMessage: true
+                })),
+                AbilityHelper.immediateEffects.discardCardsFromHand((context) => ({
```

Comment:
nit: can we get a space in between these abilities for readability

Reply:
I'm not sure where you want the space to go exactly?

Reply:
Oh sorry I just meant a newline

Reply:
added